### PR TITLE
Add GraphQL Changes to 2201.5.0 Release Note

### PR DESCRIPTION
**Pull request by DimuthuMadushan**

### CODE REVIEW

File: downloads/swan-lake-release-notes/swan-lake-2201.5.0 /RELEASE_NOTE.md [hunk 1]
```diff
@@ -144,6 +144,8 @@ To view bug fixes, see the [GitHub milestone for 2201.5.0 (Swan Lake)](https://g
 
 - Added parallel execution support for GraphQL resolvers.
 - Allowed adding a single interceptor without creating an array.
+- Skipped additional validation for unused operations in the GraphQL document.
+- Added support for exiting the listener when panic occurred.
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.5.0 /RELEASE_NOTE.md:148]:
```suggestion
- Added support for exiting the listener when a panic occurrs.
```

**Review by praneesha — APPROVED** [review]

File: downloads/swan-lake-release-notes/swan-lake-2201.5.0 /RELEASE_NOTE.md [hunk 2]
```diff
@@ -144,6 +144,8 @@ To view bug fixes, see the [GitHub milestone for 2201.5.0 (Swan Lake)](https://g
 
 - Added parallel execution support for GraphQL resolvers.
 - Allowed adding a single interceptor without creating an array.
+- Skipped additional validation for unused operations in the GraphQL document.
+- Added support for exiting the listener when a panic occurs.
```
Comment (ThisaruGuruge) [inline downloads/swan-lake-release-notes/swan-lake-2201.5.0 /RELEASE_NOTE.md:148]: This wording does not express the idea correctly. This sounds like a feature, but it is not. It's a change of behavior. @praneesha can we re-word this?

Comment (praneesha) [reply downloads/swan-lake-release-notes/swan-lake-2201.5.0 /RELEASE_NOTE.md:148]: Shall we say `Improved the listener behavior to exit when a panic occurrs.`?

Comment (ThisaruGuruge) [reply downloads/swan-lake-release-notes/swan-lake-2201.5.0 /RELEASE_NOTE.md:148]: +1

**Review by ThisaruGuruge — APPROVED** [review]

**Pull request merged**